tests/resources/Group23-VIC-Machine-Service-Util.robot

```diff
@@ -150,3 +150,11 @@ Verify VCH List Empty
     ${vchs}=  Run  echo '${OUTPUT}' | jq -r '.vchs[]'
     Log  ${vchs}
     Length Should Be  ${vchs}  0
+
+Get Docker Params API
+    [Arguments]    ${vch}    ${vch_name}
+    Get Path Under Target    ${vch}
+    ${docker_host}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${vch_name}").docker_host'
+    Set Test Variable    ${docker_host}
```

Comment:
nit-pick: maybe put this as the last line of the keyword? (I think the overall flow of: (1) get a value, (2) validate the value, (3) set the value is a little more intuitive than (1) get, (2) set, (3) validate.)

Reply:
I did this to cover the case where this test/check fails and I need the `${docker_host}` var set to any value (null in this case, maybe) so the next "back-up" check gets triggered.  I had it set the way you suggested before and the back-up check wouldn't get triggered because it would fail out.

Reply:
from readability perspective is it easier to see what's going on if we set ${docker_host} to nil first and then set it again after validation check? Or we could probably do `Run keyword If Test Failed  Set Test Variable  ${docker_host}  nil`

Reply:
The `Run    echo` command will return `null` if nothing is found by the query check.  I would have to change the retry logic to make `Run keyword If Test Failed Set Test...` work here from the keyword that calls this keyword.